Speech to Text/MicrophoneBasicViewController.swift

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright IBM Corporation 2016
```

Comment:
I guess this is mostly a rename of `MicrophoneViewController.swift`, so here we might want
> Copyright IBM Corporation 2016, 2018